docs/publish/publish.md

```diff
@@ -11,7 +11,7 @@ You can use one of several methods to deploy your Office Add-in for testing or d
 
 |**Method**|**Use...**|
 |:---------|:------------|
-|[Sideloading](../testing/create-a-network-shared-folder-catalog-for-task-pane-and-content-add-ins.md)|As part of your development process, to test your add-in running on Windows, iPad, Mac, or in a browser.|
+|[Sideloading](../testing/create-a-network-shared-folder-catalog-for-task-pane-and-content-add-ins.md#sideload-your-add-in)|As part of your development process, to test your add-in running on Windows, iPad, Mac, or in a browser.|
```

Comment:
I would recommend the following change instead and it won't be confused with the Windows-only article.
```suggestion
|[Sideloading](../testing/test-debug-office-add-ins.md#sideload-an-office-add-in-for-testing)|As part of your development process, to test your add-in running on Windows, iPad, Mac, or in a browser.|
```

Reply:
> I would recommend the following change instead and it won't be confused with the Windows-only article.

This is perfect. Thank you